docs/en/managing-data/updates.md

```diff
@@ -76,6 +81,51 @@ Note that for lightweight updates, a mutation is still used to update the data;
 
 Read more about [lightweight updates](/en/guides/developer/lightweight-update).
 
+## Collapsing Merge Tree
+
+Stemming from the idea that updates are expensive but inserts can be leveraged to perform updates,
```

Comment:
is there a reason you use line breaks?

Reply:
@gingerwizard are you referring to the edits to the query results here?

Reply:
ah, you're probably referring to the line breaks in the text itself? If so it comes from the CLion plugin I am using, it suggests where to place a line break such that the markdown is readable. space + /n will push the text to a new line, but /n without a space before it will still render on the same line, but looks more readable in the document itself when editing.